sections/06_discussion.md

```diff
@@ -25,31 +25,46 @@ human-level performance is irrelevant.*
 
 ### Interpretation
 
-As the challenge of interpretability is common across many domains, there is
-significant interest in developing generic procedures for knowledge extraction
-from deep models. Ribeiro et al [@tag:Ribeiro2016_lime] focus on interpreting
-individual predictions rather than interpreting the model. By fitting simple
-linear models to mimic the predictions of the deep learning model in a small
-neighborhood of a data sample, they generated an interpretable model for each
-prediction. While this procedure can provide interpretable models for each
-sample, it is unclear whether these interpretable models are reliable.
-Theoretical guarantees on the curvature of the predictions of deep learning
-models are not known, and it is unclear whether predictions from deep learning
-models are robust to sample noise. Toward quantifying the uncertainty of
-predictions, there has been a renewed interest in confidence intervals for
-deep neural networks. Early work from Chryssolouris et al
-[@tag:Chryssolouris1996_confidence] provided confidence intervals under the
- assumption of normally distributed error. However, Nguyen et al
- [@tag:Nguyen2014_adversarial] showed that the confidence of convolutional
- neural networks is not reliable; they can output confidence scores over
- 99.99% even for samples that are purely noise. Recently, Fong and Vedaldi
- [@tag:Fong2017_perturb] provided a framework for understanding black box
- algorithms by perturbing input data.
-
-For domain-specific models, we previously described approaches for the
-interpretation and visualization of neural networks that prediction
-transcription factor binding [@tag:Alipanahi2015_predicting
-@tag:Lanchantin2016_motif @tag:Shrikumar2016_blackbox]. Other studies have
+Approaches for understanding the patterns learned by deep learning models can be
+broadly divided into: sample-specific importance scoring, visualization 
+
+Sample-specific importance scoring and interpretation:
+- perturbation-based approaches where you observe the impact of the output on
+perturbed versions of the input and use that to ascribe importance. Simplest
+version is forward propagations (ISM, basset=filter nullification). LIME builds
+a linear model to approximate the output on the perturbed input and uses this
+to ascribe feature importance. Prediction difference analysis uses conditional
+probability to intelligently sample the perturbations. These approaches are
+computationally expensive (copy whatever the 'meaningful perturbations' paper
+said about LIME). The meaningful perturbations paper uses gradient descent
+to identify which portion of an image to blur, thereby alleviating but not completely
+eliminating the computational burden.
+- backpropagation approaches to alleviate computational burden. Simonyan,
+LRP which is gradient*input, guided backprop (kicks out negative gradients),
```

Comment:
gradient-prop

Appreciate RSI has got in the way. I'll stop correcting typos at this point, although they'll have to be attended to sometime.